### ISSUE TITLE
Windows 10 TP support on Intel WDDM2.0 drivers?

### ISSUE THREAD

**Issue by oscarbg** [01org/XenGT-Preview-xen#22]
Hi,
just asking if XenGT supports Windows 10 TP with included new WDDM 2.0 drivers (Intel HD & Iris Graphics Drivers Version 10.18.15.4098 WHQL)?
thanks..

**Comment by jikesong** [comment]
Yes. GVT-g provides a vGPU which is below the framework, so I'm pretty sure it won't matter.

**Comment by jbbr** [comment]
I tried to upgrade my windows 8 VM (with working XenGT, 1 Monitor DP) to windows 10. After reboot I only get a black screen in VNC/SDL and no output on monitor
Same with fresh install of windows 10. 

XenGT support seems to be broken/not included in Windows 10 drivers

**Comment by jikesong** [comment]
For XenGT, win10 is not validated yet.
